src/App.js

```diff
@@ -8,9 +8,7 @@ import AddBook from "./components/AddBook";
 import { useState } from "react";
 
 function App() {
-  const [claimedUrl, setClaimedUrl] = useState(
-    "https://book-swap-api.dev.io-academy.uk/api/books?claimed=0"
-  );
+  const [claimedUrl, setClaimedUrl] = useState(0);
```

Comment:
You're not actually using this anymore. Get rid of it and remove it from the nav as well